tests/integration/backend/test_backend.py

```diff
@@ -4,7 +4,12 @@
 import json
 
 from imars3d.backend.workflow.engine import WorkflowEngineAuto
+from imars3d.backend.workflow.validate import JSONValidationError
 import pytest
+from pathlib import Path
+
+
+JSON_DATA_DIR = Path(__file__).parent.parent.parent / "data" / "json"
```

Comment:
Can we make this a pytest fixture ([example](https://code.ornl.gov/sns-hfir-scse/sans/sans-backend/-/blob/next/tests/conftest.py#L28))? It should still be a `pathlib.Path`, but I imagine a ton of these tests would use this. It may be better as a separate PR.

Reply:
Yeah, I actually got this from talking with Cole, and I know he has it in his branch too.

Reply:
I could do it as part of this pr if you want

Reply:
Let's do that in a separate PR